schema.sql

```diff
@@ -38,3 +38,42 @@ ADD CONSTRAINT fk_owners
 FOREIGN KEY(owners_id)
 REFERENCES owners(id)
 ON DELETE CASCADE;
+
+-- Create a table named vets
+CREATE TABLE vets (
+ id SERIAL PRIMARY KEY,
+ name VARCHAR(200),
+ age INT,
+ date_of_graduation DATE
+);
+
+-- Create a joining between species and vet 
+CREATE TABLE specializations (
+  species_id INT NOT NULL,
+  vets_id INT NOT NULL,
+  PRIMARY KEY(species_id, vets_id),
+  CONSTRAINT fk_species
+   FOREIGN KEY(species_id)
+    REFERENCES species(id)
+     ON UPDATE CASCADE,
+  CONSTRAINT fk_vets
+   FOREIGN KEY(vets_id)
+    REFERENCES vets(id)
+     ON UPDATE CASCADE
+);
+
+-- Joining a joining between animals and vets
+CREATE TABLE visits (
+  animals_id INT NOT NULL,
+  vets_id INT NOT NULL,
+  date_of_visits DATE,
+  PRIMARY KEY(animals_id, vets_id, date_of_visits),
```

Comment:
- [ ] When you make these two Primary Keys, it will not possible to add the same vet or the same animal twice. Please drop the PKs.

Reply:
Thank you very much @NduatiKagiri . I have removed the primary key. 👍